### PR TITLE
9693 lint flow

### LINKED ISSUE #9692: Document a best practice to have modules dedicated to contain only code for orchestration flows
## What is the Problem Being Solved?

Without E support in asyncFlow, it's a best practice to have modules dedicated to contain only code for orchestration flows, we (OpCo) should document this behavior so developer can follow this convention.

### PR DESCRIPTION
**Pull request by turadg**
closes: #9692, #9693

## Description

Sets up a convention for `flows` modules and document in the orchestration README. 

Also makes lint config to test them. The lint config is exported as a shared config that developers can use. I've included it in the same PR to make sure it works with the new convention.

### Security Considerations
none

### Scaling Considerations
none

### Documentation Considerations
Documented the convention in the README. I expect it'll make its way to docs.agoric.com as needed.

### Testing Considerations
nothing special

### Upgrade Considerations
not yet deployed

### CODE REVIEW

File: packages/orchestration/src/examples/sendAnywhere.contract.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Fail } from '@endo/errors';
 import { E } from '@endo/far';
 import { M } from '@endo/patterns';
 import { withOrchestration } from '../utils/start-helper.js';
-import { orchestrationFns } from './sendAnywhereFlows.js';
+import * as flows from './sendAnywhere.flows.js';
```
Comment (dckc) [inline packages/orchestration/src/examples/sendAnywhere.contract.js:8]: :+1:

File: packages/orchestration/src/examples/sendAnywhere.contract.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const contract = async (
   );
 
   // orchestrate uses the names on orchestrationFns to do a "prepare" of the associated behavior
-  const orchFns = orchestrateAll(orchestrationFns, {
+  const orchFns = orchestrateAll(flows, {
```
Comment (dckc) [inline packages/orchestration/src/examples/sendAnywhere.contract.js:79]: :+1:

**Review by dckc — APPROVED** [review]
nice!

File: packages/orchestration/src/examples/sendAnywhere.flows.js
Comment (mhofman) [inline packages/orchestration/src/examples/sendAnywhere.flows.js:1]: Nit: git is not catching this as a rename. Could you split this commit in 2 with minimal changes to the source during the file rename?

Comment (turadg) [reply packages/orchestration/src/examples/sendAnywhere.flows.js:1]: because you want to preserve the git blame on the lines?

Comment (turadg) [reply packages/orchestration/src/examples/sendAnywhere.flows.js:1]: done, but I'll say it didn't seem worth the time to me. the reindenting already touches all the lines.

File: packages/orchestration/src/examples/auto-stake-it.contract.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ import { preparePortfolioHolder } from '../exos/portfolio-holder-kit.js';
  */
 
 /**
+ * @satisfies {OrchestrationFlow}
```
Comment (dckc) [inline packages/orchestration/src/examples/auto-stake-it.contract.js:38]: :+1:

File: packages/orchestration/test/examples/bad.flows.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @file Module with linting errors, to verify the linting config detects them.
```
Comment (dckc) [inline packages/orchestration/test/examples/bad.flows.js:2]: :clap:

File: packages/orchestration/README.md [hunk 5]
```diff
@@ -3,3 +3,18 @@
 Build feature-rich applications that can orchestrate assets and services across the interchain.
 
 Usage examples can be found under [src/examples](https://github.com/Agoric/agoric-sdk/tree/master/packages/orchestration/src/examples). They are exported for integration testing with other packages.
+
+## Orchestration flows
```
Comment (dckc) [inline packages/orchestration/README.md:7]: only skimmed this

**Review by dckc — APPROVED** [review]

**Pull request merged**